cpp/demo/hyperelasticity/hyperelasticity.ufl

```diff
@@ -59,13 +59,22 @@ lmbda = E*nu/((1 + nu)*(1 - 2*nu))
 psi = (mu/2)*(Ic - 3) - mu*ln(J) + (lmbda/2)*(ln(J))**2
 
 # Total potential energy
-Pi = psi*dx # - inner(B, u)*dx - inner(T, u)*ds
+Pi = psi*dx  # - inner(B, u)*dx - inner(T, u)*ds
 
 # First variation of Pi (directional derivative about u in the direction of v)
-F = derivative(Pi, u, v)
+F_form = derivative(Pi, u, v)
 
 # Compute Jacobian of F
-J = derivative(F, u, du)
+J_form = derivative(F_form, u, du)
+
+# Compute von Mises stress in current configuration
+sigma = (1/J)*diff(psi, F)*F.T
+sigma_deviatoric = sigma - (1 / 3) * tr(sigma) * I
+sigma_von_mises = sqrt((3 / 2) * inner(sigma_deviatoric, sigma_deviatoric))
+
+forms = [F_form, J_form]
+elements = [(element)]
+expressions = [(sigma_von_mises, [[0.25, 0.25, 0.25]])]
```

Comment:
As mentioned in our Slack discussion, I think it would be even better to use this demo to highlight that we can do interpolation into suitable function spaces.

Rephrasing this as:
```python
 expressions = [(sigma_von_mises, TensorElement("Discontinous Lagrange", "DG", (3,3))]
 ```
and using the data from: https://github.com/FEniCS/dolfinx/pull/1889
to be able to visualize the stress on the whole mesh without a projection

Reply:
Agreed, will change. In the meantime we've discovered that FFCx cannot support forms, elements and expressions simultaneously in it's output, so we'll change that first.

Reply:
Processing of multiple different objects is now fixed. I'd prefer to go with @jorgensd 's suggestion in a separate PR. We should have this PR merged asap.

Reply:
Thats fine